packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -365,24 +371,44 @@ class C4DMastheadL1 extends HostListenerMixin(StableSelectorMixin(LitElement)) {
       return markup;
     };
 
-    if (cta && cta?.title) {
-      if (cta?.ctaType === CTA_TYPE.CHAT) {
+    /**
+     * Maps CTA_TYPE values to CMApp CTA options.
+     *
+     * @see https://github.ibm.com/live-advisor/cm-app/blob/master/docs/cm-doc.md#calls-to-action
+     */
+    const iconMap = new Map([
+      [L1_CTA_TYPES.CONTACT_US, Chat16()],
+      [L1_CTA_TYPES.CHAT_NOW, Chat16()],
+      [L1_CTA_TYPES.EMAIL_US, Email16()],
+      [L1_CTA_TYPES.CALL_US, Phone16()],
+      [L1_CTA_TYPES.BOOK_A_CONSULTATION, Calendar16()],
+      [L1_CTA_TYPES.REQUEST_A_DEMO, Demo16()],
+      [L1_CTA_TYPES.REQUEST_A_QUOTE, Quote16()],
+    ]);
+
+    if (cta) {
+      const { ctaType } = cta;
+      if (ctaType && iconMap.has(ctaType as L1_CTA_TYPES)) {
+        const icon = iconMap.get(ctaType as L1_CTA_TYPES);
         return html`
           <button
             class="${classname}"
-            data-ibm-contact="contact-link"
+            data-ibm-contact="${ctaType}-link"
             aria-label="${ifDefined(contactCtaLabel)}">
             ${desktopWrapper(
-              html`<span data-ibm-contact="contact-text">${cta.title}</span
-                >${Chat16()}`
+              html`<span data-ibm-contact="${ctaType}-text"
+                  >${cta?.title ?? ''}</span
```

Comment:
On line 405 we check to make sure we have some truthy value for cta.title before rendering a link. Should we avoid rendering text-less buttons if no title value is provided?

Reply:
Right, so I left this available in case there is some way to get the CMApp to inject the correct localized text. Better to remove altogether?